md_run_multiple.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np 
+import pickle
+import sys
+import os
+import ase.io
+from ocdata.vasp import run_vasp, write_vasp_input_files, VASP_FLAGS
+
+def get_traj_files(pklfile, index, count):
+    with open(pklfile, 'rb') as f:
+        ans = pickle.load(f)
+        return ans[index:(index+count)]
+
+
+def get_system_id(trajname):
+    return int(trajname.split("/")[-1].split(".")[0][6:])
+
+
+def process_for_md(traj_name, opdir, index):
+
+    full_traj = ase.io.read(traj_name, ":")
+    sys_id = get_system_id(traj_name)
+
+    np.random.seed(sys_id) # seed the sampler 
+    n = len(full_traj)
+    assert n > 0
+
+
+    # set number of outer loop iterations to 40
+    VASP_FLAGS['nsw'] = 40
+
+    # modify IBRION to 0 for MD
+    VASP_FLAGS['ibrion'] = 0
+
+    VASP_FLAGS['laechg'] = False # don't save CHG
+
+    # adding the following from Muhammed's INCAR file
+    VASP_FLAGS['potim'] = 2
+    VASP_FLAGS['tebeg'] = 900 # 900 K
+    VASP_FLAGS['teend'] = 900 # 900 K
+    VASP_FLAGS['mdalgo'] = 0 
+    VASP_FLAGS['smass'] =  -3
+
```

Comment:
VASP_FLAGS['lcharg'] = False
(Forgot about this, wasn't included in my INCAR)

This actually controls whether CHG files get written. laechg controls the electron density files.

Reply:
Done